src/components/App/index.tsx

```diff
@@ -11,8 +11,13 @@ import constants from '../Constants';
 
 const App = props => {
   const [healthCheck, setHealthCheck] = useState({
-    isVerified: false,
-    allowAccess: false,
+    oneauthVerified: false,
```

Comment:
on both state variables, name it as isVerified and allowAccess. Since you have two outer variables to differentiate the system, structure can be retained as same